signer/src/testing/transaction_signer.rs

```diff
@@ -217,8 +207,6 @@ where
         // integration test.
         tokio::time::sleep(Duration::from_millis(250)).await;
 
-        handle.join_handle.abort();
-
```

Comment:
qq: why don't we need those around anymore?

Reply:
Two reasons:
1. They were the cause of some tests stalling.
2. I believe that spawned tasks are cleaned up as part of test clean up.